parity.gemspec

```diff
@@ -9,7 +9,6 @@ Gem::Specification.new do |spec|
   eos
 
   spec.email = ["dan@thoughtbot.com"]
-  spec.executables = ["development", "staging", "production"]
```

Comment:
If `brew install parity` installs the executables correctly without this line, then I'm fine with it.